src/dra/5298C.c

```diff
@@ -169,8 +169,9 @@ s32 CalcAttack(s32 equipId, s32 otherEquipId) {
     if (equipId == 4 && D_800A4B04[otherEquipId].itemCategory == ITEM_SHIELD) {
         totalAttack += 5;
     }
-    if (equipId == 0x7E) {                  // Equippable Sword Familiar
-        totalAttack += g_Status.D_80097C74; // Level of sword familiar
+    if (equipId == 0x7E) { // Equippable Sword Familiar
+        totalAttack +=
+            g_Status.statsFamiliars[4].level; // Level of sword familiar
```

Comment:
enum for familiar IDs?

Reply:
Probably a good idea, but not for now for this function - Determining each familiar's ID would be its own task. I think someone working out the 3 parameters of statsFamiliar (level, probably xp, and something else), along with the ID for each familiar, would be a good task to tackle. I might do that some time soon, or someone else can if it interests them.